themes/finna2/js/finna-record.js

```diff
@@ -29,6 +29,18 @@ finna.record = (function finnaRecord() {
           description.hide();
         });
     }
+    const button = $('.show-hide-button');
+    const more = button.html();
+    const less = $('.hide-info').html();
+    $('.cc-info').on('show.bs.collapse', function changeText() {
+      $(this).parents('.fulltextField').find(button).html(less);
```

Comment:
Sori, tuli virheellinen ohje, kun en huomioinut, että näitä voi olla monta. Perutaan tämä viimeisin js-muutos.